src/components/controls/time/Time.js

```diff
@@ -25,7 +25,7 @@ var Time = React.createClass({
      * @return {string}         Timestamp in the format of HH:MM:SS
      */
     formatTime(seconds) {
-        var date = new Date(null);
+        var date = new Date(Date.UTC(2000,1,1,0,0,0,0);
```

Comment:
I'm totally in the middle of restructuring this project for better development as this was done a long time ago. It's not as smooth as it could be. Will try get that up ASAP and then tackle PR's.